docs/source/guide/get_started/quick_start_guide/cli_commands.rst

```diff
@@ -474,10 +474,10 @@ Demonstration
 .. code-block::
 
     (otx) ...$ otx demo --help
-    usage: otx demo [-h] -i INPUT --load-weights LOAD_WEIGHTS [--fit-to-size FIT_TO_SIZE FIT_TO_SIZE] [--loop] [--delay DELAY] [--display-perf] [template] {params} ...
+    usage: otx demo [-h] -i INPUT --load-weights LOAD_WEIGHTS [--fit-to-size FIT_TO_SIZE FIT_TO_SIZE] [--loop] [--delay DELAY] [--display-perf] [--save-results-to SAVE_RESULTS_TO] [template] {params} ...
```

Comment:
`save-results-to` is a good name, but what about `output`? Other CLI commands use the name `-o, --output`, so I think output is also a good idea.

Reply:
Thank you, great point

Reply:
Done